### ISSUE TITLE
Can't add API Token

### ISSUE THREAD

**Issue by joshlsullivan** [digitalocean/doctl#578]
On Windows 10, I've added `doctl` the the system path, but when I run `doctl auth init`, I get this error:

```
PS C:\Users\joshl> doctl auth init
DigitalOcean access token: Error: unable to read DigitalOcean access token: The handle is invalid.
```

**Comment by andrewsomething** [comment]
Hi @joshlsullivan A fix for this issue has been identified and merged into master. It will be available in the next release. See: https://github.com/digitalocean/doctl/pull/574

**Comment by bentranter** [comment]
@joshlsullivan This should now be fixed in [v1.31.0](https://github.com/digitalocean/doctl/releases/tag/v1.31.0).

**Comment by joshlsullivan** [comment]
Confirmed working.
